frontend/src/pages/Menu.tsx

```diff
@@ -130,22 +130,17 @@ function App() {
             </div>
           </div>
           <div className="cartFooter">
-            <div className="cartRow">
-              <Typography variant="body1">Delivery</Typography>
-              <div className="dash" />
-              <Typography variant="body1">{formatter.format(1.5)}</Typography>
-            </div>
             <div className="cartRow">
               <Typography variant="body1">Total</Typography>
               <div className="dash" />
               <Typography variant="body1">
-                {formatter.format(calculateTotal() + 1.5)}
+                {formatter.format(calculateTotal())}
               </Typography>
             </div>
-            {cart.length > 0 && (
+            {calculateTotal() > 0 && (
```

Comment:
Not really important, but doesn't it make more sense to use cart.length instead of calculateTotal() here? Otherwise you a prohibiting people from getting free stuff unless they buy something else :P

Reply:
cart.length doesn’t work for some reason and I couldn’t really debug it, so I decided to use calculated total for now